config/webpack.common.config.js

```diff
@@ -9,6 +9,12 @@ module.exports = {
     publicPath: '/',
   },
   resolve: {
+    alias: {
+      'env.config': path.resolve(process.cwd(), './env.config'),
+    },
+    fallback: {
+      'env.config': path.resolve(__dirname, './env.config.js'),
+    },
```

Comment:
I wonder if you set the `fallback` as follow,
```js
fallback: {
      'env.config': false
 },
```
Wouldn't this account for error if the file doens't exist and thus there would be neccessity for creating the empty js file?

Reply:
Nice!  That seems to work - it returns an empty object.  I'm going to have to do something for jest, since it's not using this webpack fallback stuff, but this simplifies the webpack part a bunch. Thanks, great suggestion.